src/assets/scss/global/typography.scss

```diff
@@ -29,6 +29,14 @@ h1.huge {
 	@include overlay-text();
 }
 
+.medium-text {
```

Comment:
Started seeing patterns with the text sizes in Olly's new comp. Added these two mixins here and to the typography styleguide